turbo/snapshotsync/external_downloader.pb.go

```diff
@@ -7,12 +7,13 @@
 package snapshotsync
```

Comment:
You can delete this, it will now need to live in `gointerfaces` package

Reply:
I'll do it in a separate PR, it you don't mind.